app/presenters/tasks/capital_assessment.rb

```diff
@@ -22,6 +22,12 @@ def in_progress?
       capital.present?
     end
 
+    # Capital task is only complete when both the AnswersValidator
```

Comment:
This was resulting in the task list incorrectly showing the capital section as complete when the confirmation question had not been added.

Reply:
If we don't save the confirmation question value, won't this be unreliable to use?

Reply:
https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/db/schema.rb#L49